src/web/theme/pages/OurSelection/SiteCard/Sitecard.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropType from "prop-types";
+import "./SiteCard.scss";
+
+const SiteCard = ({ site }) => {
+  return (
+    <div className="site-card">
+      <div className="site-card__image">
+        <img src={site.image} />
+      </div>
+      <div className="site-card__content">
+        <div className="site-card__content__name">
+          <span>{site.name}</span>
+        </div>
+        <div className="site-card__content__description">
+          {site.description}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+SiteCard.PropType = {
+  site: PropType.object
```

Comment:
Nit : It would be best to describe the shape of the object :slightly_smiling_face: